kubernetes/loculus/values.yaml

```diff
@@ -131,6 +135,12 @@ defaultOrganismConfig: &defaultOrganismConfig
             timestamp: ncbiReleaseDate
         noInput: true
         columnWidth: 100
+      - name: earliestReleaseDate
```

Comment:
This should go into the default metadata part of the helm template as it's a magic value and not something that's calculated by preprocessing